pkg/fanal/analyzer/sbom/sbom.go

```diff
@@ -0,0 +1,80 @@
+package sbom
+
+import (
+	"context"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/exp/slices"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/sbom"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&sbomAnalyzer{})
+}
+
+const version = 1
+
+var requiredSuffixes = []string{
+	".spdx",
+	".cdx",
+}
+
+type sbomAnalyzer struct{}
+
+func (a sbomAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
+	// Format auto-detection
+	format, err := sbom.DetectFormat(input.Content)
+	if err != nil {
+		return nil, xerrors.Errorf("failed to detect SBOM format: %w", err)
+	}
+
+	bom, err := sbom.Decode(input.Content, format)
+	if err != nil {
+		return nil, xerrors.Errorf("SBOM decode error: %w", err)
+	}
+
+	// For Bitnami images
+	if strings.HasPrefix(input.FilePath, "opt/bitnami/") {
```

Comment:
Maybe we need to add generic logic for other folders.

But we can do it after user questions.

Reply:
Yes, agree. But I guess there is no standard for this.

Reply:
I am referring to this discussion: https://github.com/aquasecurity/trivy/pull/4210/files#r1212955004

Even in current `main`, the `sbom.go` only checks the hardcoded path `/opt/bitname` for spdx-SBOMs:

```go
// For Bitnami images
	if strings.HasPrefix(input.FilePath, "opt/bitnami/") {
```

Or am I reading this wrong and other paths/formats are somehow implicitly supported?